alibuild_helpers/build.py

```diff
@@ -91,36 +91,47 @@ def __init__(self, remoteStore, architecture, workdir, insecure):
     self.httpBackoff = 0.4
     self.doneOrFailed = []
 
-  def getRetry(self, url, dest=None):
+  def getRetry(self, url, dest=None, returnResult=False):
```

Comment:
This is fine for now, however we should probably simply rely on requests rather than implement our own logic.